libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectGeometry.kt

```diff
@@ -65,10 +63,9 @@ class RouteAlertGeometry private constructor(
      */
     override fun hashCode(): Int {
         var result = length.hashCode()
-        result = 31 * result + startCoordinate.hashCode()
-        result = 31 * result + startGeometryIndex
-        result = 31 * result + endCoordinate.hashCode()
-        result = 31 * result + endGeometryIndex
+        result = 31 * result + shape.hashCode()
+        result = 31 * result + (startGeometryIndex ?: 0)
+        result = 31 * result + (endGeometryIndex ?: 0)
```

Comment:
```suggestion
        result = 31 * result + startGeometryIndex.hashCode()
        result = 31 * result + endGeometryIndex.hashCode()
```

Reply:
it's auto-generated, I think we should not change it